modules/articles/server/controllers/articles.server.controller.js

```diff
@@ -81,14 +81,45 @@ exports.delete = function (req, res) {
  * List of Articles
  */
 exports.list = function (req, res) {
-  Article.find().sort('-created').populate('user', 'displayName').exec(function (err, articles) {
-    if (err) {
-      return res.status(400).send({
-        message: errorHandler.getErrorMessage(err)
-      });
-    } else {
-      res.json(articles);
-    }
+  // Prepare paging parameters
+  var page = Math.max(0, req.query.page - 1); // using a zero-based page index for use with skip()
+  var take = req.query.take || 10;
+
+  /** 
+  * I don't like having the filter, so I'm removing it for now. It's going to introduce a bit of complexity. 
```

Comment:
I left this comment here, so there is some context of why the filter is there & what the implications of using it are.